packages/aws-cdk-lib/aws-autoscaling/lib/auto-scaling-group.ts

```diff
@@ -575,6 +575,45 @@ export interface LaunchTemplateOverrides {
   readonly weightedCapacity?: number
 }
 
+/**
+ * InstanceMaintenancePolicy allows you to configure an instance maintenance policy for your Auto Scaling group to
+ * meet specific capacity requirements during events that cause instances to be replaced, such as an instance
+ * refresh or the health check process.
+ *
+ * https://docs.aws.amazon.com/autoscaling/ec2/userguide/ec2-auto-scaling-instance-maintenance-policy.html
+ */
+export interface InstanceMaintenancePolicy {
+  /**
+   * Specifies the upper threshold as a percentage of the desired capacity of the Auto Scaling group.
+   * It represents the maximum percentage of the group that can be in service and healthy, or pending,
+   * to support your workload when replacing instances.
+   *
+   * Value range is 100 to 200. After it's set, a value of -1 will clear the previously set value.
+   *
+   * Both `minHealthyPercentage` and `maxHealthyPercentage` must be specified, and the difference between
+   * them cannot be greater than 100. A large range increases the number of instances that can be
+   * replaced at the same time.
+   *
+   * @default - no value.
+   */
+  readonly maxHealthyPercentage?: number;
```

Comment:
```suggestion
   */
  readonly maxHealthyPercentage: number;
```
I think we can simplify the validation logic and improve user experience by setting both attributes as required.